packages/react/src/Tooltip/Tooltip.features.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import type {Meta} from '@storybook/react'
+import {Button, Box, Link, Stack} from '..'
+import {Tooltip} from './Tooltip'
+import {BookIcon} from '@primer/octicons-react'
+
+export default {
+  title: 'Components/Tooltip/Features',
+  component: Tooltip,
+} as Meta<typeof Tooltip>
+
+export const AnchorHasMargin = () => (
+  <Box padding="spacious">
+    <Tooltip text="Tooltip is still centered">
+      <Button sx={{marginLeft: 3}}>Button has 16px margin Left</Button>
```

Comment:
Brand components don't have `sx` prop support, so I'm guessing that this example isn't working as expected?

To fix this, you can add a new stylesheet called `Tooltip.stories.module.css` and apply the override style using CSS modules.

Reply:
I just used the `style` prop since this is just used to demonstrate how Tooltip alignment doesn't break.